server/src/test/YmlToBdlVisitorImpl.test.ts

```diff
@@ -0,0 +1,143 @@
+import * as assert from "assert";
+import { ANTLRInputStream, CommonTokenStream, CommonToken } from "antlr4ts";
+import { YmlToBdlLexer } from "../YmlToBdlLexer";
+import { YmlToBdlParser } from "../YmlToBdlParser";
+import { YmlToBdlVisitorImpl } from "../YmlToBdlVisitorImpl";
+import { CompletionItem } from "vscode-languageserver";
+
+describe("Extension Server Tests", () => {
+  describe("YmlToBdlVisitorImpl", () => {
+    it("should be OK to instantiate YmlToBdlVisitorImpl", done => {
+      let inputStream = new ANTLRInputStream("");
+      let lexer = new YmlToBdlLexer(inputStream);
+      let tokenStream = new CommonTokenStream(lexer);
+      let parser = new YmlToBdlParser(tokenStream);
```

Comment:
I was wondering whether using `const` around here for these things would be smart. Maybe it would make the flow slightly more evident. Dunno. TS noob.

Reply:
(same kind of stuff further down in the diff)